README.md

```diff
@@ -1,73 +1,5 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# FCC Test Suite Template
 
-# Project Name
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
```

Comment:
- [ ] Mention languages and technologies used in this project it looks professional.